sdk/resourcemanager/Azure.ResourceManager.Core/src/GenericResourceOperations.cs

```diff
@@ -24,7 +24,9 @@ public class GenericResourceOperations : ResourceOperationsBase<GenericResource>
         internal GenericResourceOperations(ResourceOperationsBase operations, ResourceIdentifier id)
             : base(operations, id)
         {
-            _apiVersion = ClientOptions.ApiVersions.TryGetApiVersion(id.Type.ToString());
+            string val;
```

Comment:
doesn't this change introduce api call in the constructor?

Reply:
No